src/utils/constants/staking.ts

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I don't think this should be here - all other text values are in the translations file, and you can pass in the symbol as a variable to the translation. Could this be moved to the translations file please?

Reply:
Done via https://github.com/interlay/interbtc-ui/pull/196/commits/c7b8792df2b80b0bb5ea0945f29de64be45e7a4b.